TrainingTests/Domain/UseCases/GetEventsUseCaseMock.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  GetEventsUseCaseMock.swift
+//  TrainingTests
+//
+//  Created by Ong Yue Huei on 19/07/2021.
+//
+
+import Combine
+@testable import Training
+
+final class GetEventsUseCaseMock: GetEventsUseCase {
+    enum CallArg: Equatable {
+        case perform(page: Int?)
```

Comment:
[nits]
出来るのであれば`let`, `required`(`not optional`)にするのが良いです！

```suggestion
        case perform(page: Int)
```

Reply:
はい、わかりました！教えていただきありがとうございます！